packages/plugin-ma-tracking/rollup.config.js

```diff
@@ -0,0 +1,93 @@
+import typescript from '@rollup/plugin-typescript';
+import commonjs from '@rollup/plugin-commonjs';
+import resolve from '@rollup/plugin-node-resolve';
+import { terser } from 'rollup-plugin-terser';
+import gzip from 'rollup-plugin-gzip';
+import execute from 'rollup-plugin-execute';
+import { exec } from 'child_process';
+
+const amplitudeSnippet = () => {
```

Comment:
Do we need this?

Reply:
We might still need it for snippet because we need another script for plugin.

Reply:
I will remove this file for now. Will add it back in separate PR.